doc/getting_started/index.ipynb

```diff
@@ -0,0 +1,808 @@
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bd84ab5-2054-4ec6-ae6c-74037c7744c7",
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "np.random.seed(1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "18ba0419-86e3-479d-9224-5d3ce7bbdb35",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "idx = pd.date_range('1/1/2000', periods=1000)\n",
+    "df = pd.DataFrame(np.random.randn(1000, 4), index=idx, columns=list('ABCD')).cumsum()\n",
+    "df.head(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95adfc83-7b39-463b-80a8-bcd801563146",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "The most convenient way to use hvPlot is to register the `.hvplot` *accessor* on the data type you are working with. This is done with a special import of the form `import hvplot.<data library>`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebaff587-72d8-48be-8c77-47e7f40c37d6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1788969d-0b81-4cf0-b53a-594cc3f0aa63",
+   "metadata": {},
+   "source": [
+    "In addition to registering the `.hvplot` accessor on Pandas objects, the import above sets the Bokeh plotting library as the default one and loads its corresponding extension.\n",
+    "\n",
+    ":::{attention}\n",
+    "In a notebook, loading the extension means that there's actually some front-end code that is injected in the cell output of the import, this code being required for HoloViews plots to behave correctly. So make sure not to remove this cell!\n",
+    ":::\n",
+    "\n",
+    "Now simply call `.hvplot()` on the DataFrame as you would call Pandas' `.plot()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cc71994c-52ec-46d1-a220-c6ad6c1ecb21",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "first_plot = df.hvplot()\n",
+    "first_plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e587cdf0-28d0-498d-bd75-4feabd9eb262",
+   "metadata": {},
+   "source": [
+    "The same process can be applied to other libraries, we'll just show another example with Xarray."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d7c6377-33dd-44a2-9dc4-8670107ef0fb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import hvplot.xarray  # noqa\n",
+    "import xarray as xr\n",
+    "\n",
+    "air_ds = xr.tutorial.open_dataset('air_temperature').load()\n",
+    "air_ds['air'].isel(time=slice(0, 1000, 60)).hvplot.image(dynamic=False, width=400, height=300)"
```

Comment:
Can you check how the plots that set `width=...` look like on a smartphone screen? I'd be great if you at least make the Getting Started look good on smartphones.

Reply:
For the plots that have a widget (e.g, using `groupby`), setting `responsive=True` was not good on smartphone screens as the widgets maintained their original size while the plots resized. This made the widget to cut across the whole plot. So I maintained the set widths.

For others, I set `responsive=True` and it was OK on both laptop and phone screens.

However, the `explorer` plot did not look good on a smartphone screen whether set to responsive or not.

Reply:
> This made the widget to cut across the whole plot

Ugh! Can you file an issue for this? That's concerning.

Reply:
> However, the explorer plot did not look good on a smartphone screen whether set to responsive or not.

Yep not too surprised.

Reply:
Filed the Issue https://github.com/holoviz/hvplot/issues/1484